src/huggingface_hub/keras_mixin.py

```diff
@@ -38,24 +40,28 @@ def _extract_hyperparameters_from_keras(model):
 
 
 def _parse_model_history(model):
-    logs = model.history.history
-    num_epochs = len(logs["loss"])
-    lines = []
-
-    for value in range(num_epochs):
-        epoch_dict = {
-            log_key: log_value_list[value] for log_key, log_value_list in logs.items()
-        }
-        values = dict()
-        for k, v in epoch_dict.items():
-            if k.startswith("val_"):
-                k = "validation_" + k[4:]
-            elif k != "epoch":
-                k = "train_" + k
-            splits = k.split("_")
-            name = " ".join([part.capitalize() for part in splits])
-            values[name] = v
-        lines.append(values)
+    lines = None
+    if model.history is not None:
+        if model.history.history != {}:
```

Comment:
Discussion: do we want to print some info to the logger about this case? Not too loudly, ofc, especially if it tends to be a common case. CC @osanseviero

Reply:
I don't think that's a super valuable addition, but maybe some logging warning is in place for this. 

Btw this seems more related to the parent PR https://github.com/huggingface/huggingface_hub/pull/679/files, it seems the diff is taken an outdated version of that branch.